twisted/internet/defer.py

```diff
@@ -1650,9 +1650,41 @@ def _tryLock():
 
 
 
+def sleep(time, result=None, reactor=None):
```

Comment:
Suggestion: In some places this is refered to as a clock, since ITimerWhatever is the only interface actually used.